### ISSUE TITLE
Color Accessibility on TimeSlotInput

### ISSUE THREAD

**Issue by KyoshiNoda** [KyoshiNoda/ScheduleFinder#215]
### **Description** 
Currently the `TimeSlotInput.tsx` doesn't have keyboard accessibility for Colors. Please fix this

### **Acceptance Criteria**

- [ ] Users should be able to tab through the colors
- [ ] #219
